index.md

```diff
@@ -308,6 +330,10 @@ Also, and let me quote [Harry Roberts](https://csswizardry.com) once again, **pr
 
 If you ask me, the very first thing a styleguide should do is describe the usual guidelines to write code.
 
+> > his
+>
+> Please consider gender-neutral language here. They/them/their is a perfectly reasonable set of singular gender-neutral pronouns. Little things like this can go a long way towards making a community feel more or less open to women and non-binary people.
```

Comment:
"They /ðeɪ/ is the third-person plural personal pronoun (subjective case)
in Modern English."

http://en.m.wikipedia.org/wiki/They

Several dictionaries had the same reference, but Wikipedia sums it up
better with a chart explaining the difference. Where has they been defined
as singular, it sounds awkward to me.
On Wed, Jan 7, 2015 at 8:32 PM Natalie Weizenbaum <notifications@github.com>
wrote:

> In index.md
> <https://github.com/HugoGiraudel/sass-guidelines/pull/63#discussion-diff-22632254>
> :
>
> > @@ -308,6 +330,10 @@ Also, and let me quote [Harry Roberts](https://csswizardry.com) once again, **pr
> >
> >  If you ask me, the very first thing a styleguide should do is describe the usual guidelines to write code.
> >
> > +> > his
> > +>
> > +> Please consider gender-neutral language here. They/them/their is a perfectly reasonable set of singular gender-neutral pronouns. Little things like this can go a long way towards making a community feel more or less open to women and non-binary people.
>
> "He or she" excludes non-binary people, in addition to sounding awkward.
> Singular "they" has been used for a long time, is in wide use today in
> casual speech, and is gaining more ground even in prescriptivist style
> guides.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HugoGiraudel/sass-guidelines/pull/63/files#r22632254>.
>